kf/nfconfig.go

```diff
@@ -42,9 +43,44 @@ type NFConfigs struct {
 	// keep track of interfaces
 	ifaces map[string]string
 
+	TestNewBpfProgram   func(ctx context.Context, program models.BPFProgram, conf *config.Config) models.BPF
+	TestLoadRootProgram func(ifaceName string, direction string, progType string, conf *config.Config) (models.BPF, error)
+
 	mu *sync.Mutex
 }
 
+var (
+	// errUnknownHostname indicates that the hostname is invalid
+	errUnknownHostName = errors.New("unknown hostname")
```

Comment:
"unknown" and "invalid" mean different things.  If you mean unknown, then update line 53 to not use the word "invalid".  If you mean invalid, then update both lines to not use the word "unknown".